src/dfastbe/support.py

```diff
@@ -1884,9 +1879,9 @@ def get_banklines(sim: SimulationObject, h0: float) -> geopandas.GeoSeries:
             else:
                 Lines[i] = poly_to_line(nnodes, X[i], Y[i], WET_node[i], H_node[i], h0)
     Lines = [line for line in Lines if not line is None and not line.is_empty]
-    multi_line = shapely.ops.cascaded_union(Lines)
-    merged_line = shapely.ops.linemerge(multi_line)
-    return geopandas.GeoSeries(merged_line)
+    multi_line = union_all(Lines)
+    merged_line = line_merge(multi_line)
+    return geopandas.GeoSeries(merged_line, crs="EPSG:28992")
```

Comment:
please use the crs from the configfile class , check how you can pass the crs from the configfile to here